crds/workspace.devfile.io_devworkspacetemplates.v1beta1.yaml

```diff
@@ -207,10 +207,11 @@ spec:
                       commandLine:
                         description: "The actual command-line string \n Special variables
                           that can be used: \n  - `$PROJECTS_ROOT`: A path where projects
-                          sources are mounted \n  - `$PROJECT_SOURCE`: A path to a
-                          project source ($PROJECTS_ROOT/<project-name>). If there
-                          are multiple projects, this will point to the directory
-                          of the first one."
+                          sources are mounted as defined by container component's
+                          sourceMapping \n  - `$PROJECT_SOURCE`: A path to a project
+                          source ($PROJECTS_ROOT/<project-name>). If there are multiple
+                          projects, this will point to the directory of the first
+                          one"
```

Comment:
Missing ending period. The same for the rest so just do a search and I am not marking the others.